pyvista/core/filters/poly_data.py

```diff
@@ -552,17 +555,17 @@ def curvature(self, curv_type='mean', progress_bar=False):
 
         Examples
         --------
-        Calculate the mean curvature of the hills example mesh.
+        Calculate the mean curvature of the hills example mesh and plot it.
 
         >>> from pyvista import examples
         >>> hills = examples.load_random_hills()
         >>> curv = hills.curvature()
-        >>> curv  # doctest:+SKIP
-        array([0.20587616, 0.06747695, ..., 0.11781171, 0.15988467])
+        >>> hills.plot(scalars=curv)
 
-        Plot it.
+        Show the curvature array.
 
-        >>> hills.plot(scalars=curv)
+        >>> curv  # doctest:+SKIP
+        array([0.20587616, 0.06747695, ..., 0.11781171, 0.15988467])
```

Comment:
Could you show us your reason for leaving this behind?
I assume it is a floating point problem.

Reply:
Correct. Until `doctest-modules` supports "close enough" natively at least. I know that numpy supports `may vary`, and we can consider adding that later, but it will require us to do additional work.

Reply:
doctest-modules does support close enough.  We already have it in the configuration here https://github.com/pyvista/pyvista/blob/9f5e989f2bde1c4b20d7ef6dcac039fa0dbf2c54/pyproject.toml#L159

The matching precision is based on the value written in the docstring.